### PR TITLE
Docs: Update Iceberg REST docs to reflect that view management via Iceberg views are supported

### PR DESCRIPTION
**Pull request by amogh-jahagirdar**
## Description

This is a follow on PR to https://github.com/trinodb/trino/pull/19818/files for updating docs and release notes to reflect the fact that now view management via Iceberg views is supported for the REST Catalog in Trino.

## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
(X) Release notes are required, with the following suggested text:

View management for the Iceberg REST Catalog is supported via the Iceberg view spec.

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/object-storage/metastores.md [hunk 1]
```diff
@@ -490,8 +490,10 @@ iceberg.catalog.type=rest
 iceberg.rest-catalog.uri=http://iceberg-with-rest:8181
 ```
 
-The REST catalog does not support [view management](sql-view-management) or
-[materialized view management](sql-materialized-view-management).
+The REST catalog supports [view management](sql-view-management)
```
Comment (mosabua) [inline docs/src/main/sphinx/object-storage/metastores.md:493]: Can you confirm that all these SQL command for view management in the linked section are supported please

Comment (amogh-jahagirdar) [reply docs/src/main/sphinx/object-storage/metastores.md:493]: Yes, all the SQL commands in the linked section are supported.

**Comment by devendrasr** [comment]
+1

**Review by mosabua — APPROVED** [review]
All good to go.

**Pull request merged**